src/flows/recommendation_api/corpus_candidate_sets/pockethits.py

```diff
@@ -21,13 +21,14 @@
 
 POCKETHITS_SQL = """
 SELECT
-    APPROVED_CORPUS_ITEM_EXTERNAL_ID as ID,
-    TOPIC
-FROM "SCHEDULED_CORPUS_ITEMS"
+    APPROVED_CORPUS_ITEM_EXTERNAL_ID as "ID",
+    TOPIC as "TOPIC"
+FROM "ANALYTICS"."DBT"."SCHEDULED_CORPUS_ITEMS"
 WHERE SCHEDULED_SURFACE_ID = %(SURFACE_GUID)s
 AND SCHEDULED_CORPUS_ITEM_SCHEDULED_AT BETWEEN DATEADD(day, %(MAX_AGE_DAYS)s, CURRENT_DATE) AND CURRENT_DATE
 QUALIFY row_number() OVER (PARTITION BY APPROVED_CORPUS_ITEM_EXTERNAL_ID ORDER BY SCHEDULED_CORPUS_ITEM_SCHEDULED_AT DESC) = 1
 ORDER BY SCHEDULED_CORPUS_ITEM_SCHEDULED_AT DESC
+LIMIT 8  -- Only include past Pocket Hits stories if today's aren't available. There are 8 stories per email.
```

Comment:
again, we can discuss, but my view is a top8 ranker would be better.  this would allow a larger candidate set that could be reused for personalization over a set from a larger time window.  this approach is constraining the candidate set with the slate requirements.

Reply:
I view the candidate set as all items that are eligible to be shown. The goal of this candidate set is to have timely Pocket Hits stories. The `limit 8` achieves that because each day 8 Pocket Hits stories are scheduled. I agree that it would be better to not make that assumption here, and instead filter only on `scheduled_at`.